events/models.py

```diff
@@ -53,3 +53,13 @@ class EventParticipant(TimeStampedModel):
 
     def __str__(self):
         return self.student_name
+        
+class requestEvent(models.Model):
```

Comment:
- Here also, In classes, you should follow `CamelCase` notation.
- Try to add docstrings to explain what the class/method is doing.
- You can also add docstrings for existing classes/methods.